aday2dream/src/main/kotlin/com/aday2dream/aday2dream/service/JwtService.kt

```diff
@@ -0,0 +1,58 @@
+package com.aday2dream.aday2dream.service
+
+import io.jsonwebtoken.Claims
+import io.jsonwebtoken.ExpiredJwtException
+import io.jsonwebtoken.Jwts
+import io.jsonwebtoken.SignatureAlgorithm
+import io.jsonwebtoken.security.Keys
+import io.jsonwebtoken.security.SignatureException
+import org.springframework.security.core.userdetails.UserDetails
+import java.security.Key
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+import java.util.Date
+
+object JwtService {
+
+    private val SECRET_KEY: Key = Keys.secretKeyFor(SignatureAlgorithm.HS256)
+    private const val MINUTES: Long = 60
```

Comment:
such properties need to be extracted in application.properties and then accessed using the @Value annotation in code (this is one of the ways to do it)